File: dart/tool/latn_iast_transliteration_verification/gujr_to_deva_test.dart
// ignore_for_file: avoid_print

import 'dart:convert';

import 'package:lipimala/deva_gujr_converter.dart';
import 'example_gujr.dart' as examples;

void main() {
  print('----------------------------------------------------------------');
  print(' GUJARATI TO DEVANAGARI SCRIPT CONVERSION');
  print('----------------------------------------------------------------');

  for (final source in examples.gujaratiSmokeSamples) {
    final result = source.toCanonicalDevanagariFromGujarati();
    print('"$source" -> "$result"');
  }
}

File: javascript/tools/latn_iast_transliteration_verification/gujr-to-deva-test.js
import { gujaratiSmokeSamples } from '../../src/example-gujr.js';
import { toCanonicalDevanagariFromGujarati } from '../../src/deva-gujr-converter.js';

console.log('----------------------------------------------------------------');
console.log(' GUJARATI TO DEVANAGARI SCRIPT CONVERSION');
console.log('----------------------------------------------------------------');

for (const source of gujaratiSmokeSamples) {
    const result = toCanonicalDevanagariFromGujarati(source);
    console.log(`"${source}" -> "${result}"`);
}
